lib/logger/lib/logger/utils.ex

```diff
@@ -74,6 +74,12 @@ defmodule Logger.Utils do
     truncate_n_list(list, n, [])
   end
 
+  defp truncate_n(other, _n) do
+    raise ArgumentError,
+          "cannot truncate chardata because it contains something that is not " <>
+            "valid chardata: #{inspect(other)}"
```

Comment:
I believe Logger works with iodata?

Reply:
@josevalim from these lines: https://github.com/elixir-lang/elixir/blob/5947ed751eb0276cb7fda666c319dba451a5e9e7/lib/logger/lib/logger/utils.ex#L68-L71

it looks to me like we're using chardata.

Reply:
You are correct, apologies.

Reply:
![](https://vignette.wikia.nocookie.net/es.futurama/images/9/9b/Roberto.png/revision/latest?cb=20130123221057)